### PR TITLE
global registration test

### PR DESCRIPTION
**Pull request by pondrejk**
Tests for global registration 6.9 feature:
- end-to-end test covers the registration scenario with rex and insights enabled via global parameter, rhel 7 host is registred via capsule
- additional test checks the form selections translate to curl command parameters as expected

results:
```
pytest tests/foreman/ui/test_host.py -k global_registration_end_to_end
===================================================== test session starts =====================================================
collected 32 items / 31 deselected / 1 selected                                                                               

tests/foreman/ui/test_host.py .                                                                                         [100%]

================================== 1 passed, 31 deselected, 35 warnings in 468.76s (0:07:48) ==================================

---

pytest tests/foreman/ui/test_host.py -k global_registration_form
===================================================== test session starts =====================================================
collected 32 items / 31 deselected / 1 selected                                                                               

tests/foreman/ui/test_host.py .                                                                                         [100%]

================================== 1 passed, 31 deselected, 38 warnings in 172.48s (0:02:52) ==================================
```

requires https://github.com/SatelliteQE/airgun/pull/556

### CODE REVIEW

**Comment by pondrejk** [comment]
@stejskalleos @ares feel free to take a look

File: tests/foreman/ui/test_host.py [hunk 1]
```diff
@@ -372,6 +359,37 @@ def module_libvirt_hostgroup(
     ).create()
 
 
+@pytest.fixture(scope='module')
```
Comment (swadeley) [inline tests/foreman/ui/test_host.py:362]: Hi, I see there is a `def module_manifest_org():` fixture in `pytest_fixtures/api_fixtures.py` you could try.

File: tests/foreman/ui/test_host.py [hunk 2]
```diff
@@ -372,6 +359,37 @@ def module_libvirt_hostgroup(
...
+
+
+@pytest.fixture(scope='module')
+def module_activation_key(manifest_org):
+    """Create activation key using default CV and library environment."""
+    activation_key = entities.ActivationKey(
+        auto_attach=True,
+        content_view=manifest_org.default_content_view.id,
+        environment=manifest_org.library.id,
+        name=gen_alpha(),
```
Comment (swadeley) [inline tests/foreman/ui/test_host.py:377]: I think random name is generated by default, see examples in `pytest_fixtures/api_fixtures.py`

**Review by swadeley — APPROVED** [review]
ACK

**Review by omkarkhatavkar — APPROVED** [review]
ACK

**Comment by pondrejk** [comment]
Added the insecure checkbox that came in sn 14, depends on https://github.com/SatelliteQE/airgun/pull/558

**Comment by mshriver** [comment]
trigger: test-robottelo
pytest: tests/foreman/ui/test_host.py -k global

File: tests/foreman/ui/test_host.py (hunk 1 above)
Comment (mshriver) [reply tests/foreman/ui/test_host.py:361]: At this time we're not necessarily enforcing use of the common fixtures, but it is highly encouraged to not define an equivalent entity local to the test module.

I wouldn't block merging this PR with this definition here, but this fixture is identical to the one @swadeley has pointed out.

https://github.com/SatelliteQE/robottelo/blob/master/pytest_fixtures/api_fixtures.py#L55

Given pytest fixture resolution, @pondrejk this wouldn't require any changes other than simply deleting L362-366. `manifest_org` will still be resolved by `module_activation_key`.

**Review by mshriver — APPROVED** [review]
Looks good, giving a chance for robottelo-runner to run.

**Comment by ares** [comment]
sorry for the late reply @pondrejk, that's a nice tier 3 test, do you also plan to add verification of the content access (when used together with activation key e.g.)?

**Pull request merged**